server/data/data-ldap-integration-testing/src/test/java/org/apache/james/user/ldap/ReadOnlyUsersLDAPRepositoryTest.java

```diff
@@ -37,6 +37,7 @@
     private static final String ADMIN_PASSWORD = "mysecretpassword";
     private static final String JAMES_USER = "james-user";
     private static final String JAMES_USER_MAIL = "james-user" + "@" + DOMAIN;
+    private static final String JAMES_USER_MAIL_MALICIOUS = "j*";
```

Comment:
no need to have it as a constant as it's used only once.

Reply:
Hum I don't agree with you it help explaining what "j*" is

Reply:
Moved to a local variable on the test method